tests/testthat/test.tidy_draws.R

```diff
@@ -48,6 +48,13 @@ test_that("tidy_draws works with rstanarm", {
   expect_equal(tidy_draws(m_ranef), draws_tidy)
 })
 
+test_that("tidy_draws works with rstanarm models with one variable", {
+  skip_if_not_installed("rstanarm")
+  
+  m_one_var = readRDS(test_path("../models/models.rstanarm.one_var.rds"))
```

Comment:
Can you add the code that generates this test model somewhere around here:

https://github.com/mjskay/tidybayes/blob/fb494368784350b0269fda406a37c1195d5401d6/build_test_models.R#L172

so that the test model can be regenerated in future versions of rstanarm

Reply:
Addressed in [`ca55b8b` (#333)](https://github.com/mjskay/tidybayes/pull/333/commits/ca55b8b554f65b5e09a80d33e554331f4d896c9a)

Reply:
thanks! can you put it in `build_test_models.R` in the "rstanarm" section using `saveRDS(strip_rstanarm_model(...))` like the other models in that section? I try to keep the test model files as small as possible in the repo.

Reply:
Addressed in [`6f286f7` (#333)](https://github.com/mjskay/tidybayes/pull/333/commits/6f286f7004943cd701b56c7c412e1160a12061c1)

Reply:
thanks!